opensrp-core/src/main/java/org/smartregister/repository/helper/OpenSRPDatabaseErrorHandler.java

```diff
@@ -30,5 +33,20 @@ public void onCorruption(SQLiteDatabase dbObj) {
                 Timber.e(e, "Exception closing Database object for corrupted database, ignored");
             }
         }
+
+        copyDB(dbObj.getPath());
```

Comment:
So, by default when this class is not provided, the `DefaultDatabaseErrorHandler` https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/database/DefaultDatabaseErrorHandler.java is used and it deletes the DB after closing it. Do we need to delete this also? Is there a possibility of the app not working given the corrupted DB if we do not delete it